packages/button/package.json

```diff
@@ -25,7 +25,8 @@
       "require": "./dist/umd/testing/index.js",
       "import": "./dist/esm/testing/index.js",
       "types": "./dist/types/testing/index.d.ts"
-    }
+    },
+    "./Button": "./src/Button/index.ts"
```

Comment:
I wonder if we should be explicitly listing the `.styles` file as the export?
```ts
import { PRIMARY_BUTTON_INTERACTIVE_GREEN } from '@leafygreen-ui/button/Button.styles';
```

As it is, I'd be confused why I can't do:
```ts
import { Button } from '@leafygreen-ui/button/Button';
```